frameworks/cassandra/docs/limitations.md

```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
 enterprise: 'no'
 ---
 
-* Data Center Name
+## Rack-Aware Placement
+
+The DC/OS Cassandra Service does not currently leverage rack information for placement decisions.  However, placement across racks can be performed manually using placement constraints; application-level configuration will still not be possible.   
```

Comment:
While technically correct, I can see how someone might read this and come away with an incorrect understanding. I recommend we KISS and state something like "Cassandra's Rack-Aware Replication is not supported at this time."